src/services/redisSessionStorage.ts

```diff
@@ -697,22 +671,25 @@ export default class RedisSessionStorage extends RedisStorageUtils
   private async removeOtherUserSessions(
     user: UserV3
   ): Promise<Either<Error, boolean>> {
-    const sessionInfoKeys = await this.readSessionInfoKeys(user.fiscal_code);
-    if (isRight(sessionInfoKeys)) {
-      const sessionKeys = sessionInfoKeys.value
-        .filter(
-          _ =>
-            _.startsWith(sessionInfoKeyPrefix) &&
-            _ !== `${sessionInfoKeyPrefix}${user.session_token}`
-        )
-        .map(_ => `${sessionKeyPrefix}${_.split(sessionInfoKeyPrefix)[1]}`);
+    const errorOrSessionInfoKeys = await this.readSessionInfoKeys(
+      user.fiscal_code
+    );
+    if (isRight(errorOrSessionInfoKeys)) {
+      const oldSessionInfoKeys = errorOrSessionInfoKeys.value.filter(
+        _ =>
+          _.startsWith(sessionInfoKeyPrefix) &&
+          _ !== `${sessionInfoKeyPrefix}${user.session_token}`
+      );
+      const oldSessionKeys = oldSessionInfoKeys.map(
+        _ => `${sessionKeyPrefix}${_.split(sessionInfoKeyPrefix)[1]}`
```

Comment:
may you add a comment here on what this lines do?

Reply:
Fixed in da96bdb